feat(changelog): instances-changed-windows-images-network-configuration- 2025-08-25 (#5457)
scaleway/docs-content: pages/instances/reference-content/enabling-dhcp-network-configuration-windows-server-2022.mdx

For existing running Instances, no changes to network configuration are necessary unless you plan to update the public IPv4 address. In such cases, the Instance may retain its static network configuration, continuing to use the previous public IPv4 address. To switch to DHCP and ensure the new address is applied correctly, follow the steps below to enable DHCP before assigning the new IP.

<Message type="important">
    These steps involve modifying network settings, which carry a risk of losing network connectivity (e.g., SSH or RDP access). To avoid this, enable and configure serial console access on your Instance beforehand. This allows you to maintain access via the Scaleway console even if standard network connections fail.
</Message>

<Requirements />

- A Scaleway account logged into the [console](https://console.scaleway.com)
- [Owner](/iam/concepts/#owner) status or [IAM permissions](/iam/concepts/#permission) allowing you to perform actions in the intended Organization

> - An [Instance](/instances/how-to/create-an-instance/) running Windows Server 2O22

Reviewer: ```suggestion
- An [Instance](/instances/how-to/create-an-instance/) running on Windows Server 2022
```

## Enabling serial console access

Scaleway provides a serial console (TTY) for Instances, which is essential for troubleshooting network issues.
For Windows Server Instances, you must configure the OS to enable the Special Administration Console (SAC) for serial access.

### Enabling SAC in Windows (OS Configuration)

Connect to your Windows Instance via RDP or another method and run the following commands in an elevated Command Prompt (Administrator) or PowerShell:

1. Enable Emergency Management Services (EMS):
   ```
   bcdedit /emssettings EMSPORT:1
   bcdedit /ems {current} on
   ```

2. (Optional but recommended) Enable the boot menu in the serial console:
   ```
   bcdedit /set {bootmgr} displaybootmenu yes
   bcdedit /set {bootmgr} timeout 10
   bcdedit /set {bootmgr} bootems yes
   ```

3. Reboot the Instance for changes to take effect:
   ```
   shutdown /r /t 0
   ```

> For more details, refer to [Microsoft's Azure Serial Console for Windows](https://learn.microsoft.com/en-us/troubleshoot/azure/virtual-machines/windows/serial-console-windows) documentation on enabling serial console for Windows VMs.

Reviewer: ```suggestion
For more details, refer to [Microsoft's Azure Serial Console for Windows](https://learn.microsoft.com/en-us/troubleshoot/azure/virtual-machines/windows/serial-console-windows) documentation on enabling Serial Console for Windows VMs.
```

> ### Accessing the serial console via Scaleway console

Reviewer: ```suggestion
### Accessing the Serial Console via the Scaleway console
```
